docs/Reference.md

```diff
@@ -10,6 +10,24 @@ The base URL for all API requests is:
 https://discordapp.com/api
 ```
 
+## API Versioning
+
+Discord exposes different versions of our API. You can specify version by including it in the request path:
+
+```
+https://discordapp.com/api/v{version_number}
+```
+
+Omitting the version number from the route will route requests to the current default version. API versions below 6 will be discontinued on August 16, 2017, after which they will be non-functioning. At that time, API v6 will be made the default.
```

Comment:
put a `danger` block for the discontinued info

put a `warn` block for api v6 becoming default